NSS-Bashful-Turtles/Yak-Social-Network
# Dynamic Profile Name in Navbar

Files Affected:
Navbar.js
index.js

Testing PR:
```
git fetch --all
git checkout JS-adding-dynamic-profileName

```
1. If you don't have session user. Go to index.js and comment out Router content
1. Add 
```
<Login/>
```
1. Sign in using an existing user. [user1, user2, user3, etc.] password is 'password'
1. Upon login, the nav should have the correlating users name.
1. Check with at least two logins
